docs/src/less/dox-chrome.less

```diff
@@ -213,6 +219,12 @@
 						top: @ts-unit-triple;
 					}
 				}
+				> .ts-menu > li > a {
+					padding-left: (@ts-unit / 2) * 5;
+				}
+				> .ts-menu > li > a > i {
+					padding-left: (@ts-unit / 2) * 2;
```

Comment:
this is just getting silly =) `@ts-unit`

Reply:
Fixed.